### PR TITLE
WDS-78 date picker fixes

### PR DESCRIPTION
**Pull request by Ctepan**
add custom trigger support
add model value change on every pick
add support for custom shortcuts
add using shortcut title when selected one
fix styles
add date picker type and shortcuts types to export

### CODE REVIEW

**Review by stefashkaa — APPROVED** [review]

File: .changeset/dry-weeks-compare.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/ui': patch
+---
+
+**fix**(`SDatePicker`): add custom trigger support
```
Comment (0x009922) [inline .changeset/dry-weeks-compare.md:5]: "add custom trigger support" sounds like a new feature, not a fix, isn't it?

File: .changeset/neat-squids-pretend.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/ui': patch
+---
+
+**fix**(`SDatePicker`): add support for custom shortcuts
```
Comment (0x009922) [inline .changeset/neat-squids-pretend.md:5]: Sounds like a feature.

File: .changeset/sharp-ligers-cross.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@soramitsu-ui/ui': patch
+---
+
+**fix**(`SDatePicker`): add using shortcut title when selected one
```
Comment (0x009922) [inline .changeset/sharp-ligers-cross.md:5]: Sounds like a feature.

File: .changeset/dry-weeks-compare.md (hunk 1 above)
Comment (Ctepan) [reply .changeset/dry-weeks-compare.md:5]: It was in 'documentation' but not implemented. I fixed it

File: .changeset/sharp-ligers-cross.md (hunk 3 above)
Comment (Ctepan) [reply .changeset/sharp-ligers-cross.md:5]: There is just fixed value shown

**Pull request merged**